packages/page-staking/src/Payouts/index.tsx

```diff
@@ -150,11 +150,19 @@ function getOptions (api: ApiPromise, eraLength: BN | undefined, historyDepth: B
   return [{ text: '', value: 0 }];
 }
 
+function getMyStashesIndex (api: ApiPromise, hasOwnValidators: boolean) {
+  return (isFunction(api.tx.staking.payoutStakers) && hasOwnValidators) ? 0 : 1;
+}
```

Comment:
```suggestion
function getMyStashesIndex (hasOwnValidators: boolean) {
  return hasOwnValidators ? 0 : 1;
}
```

Removing support for ancient payouts in https://github.com/polkadot-js/apps/pulls (not sure this needs to be an own function)

Reply:
yeah, no it adds no gain - will inline